.github/workflows/test.yml

```
@@ -0,0 +1,91 @@
+name: Timber starter theme tests
+
+on:
+  push:
+    branches:
+      - '1.x'
+      - '2.x'
+  pull_request:
+    types:
+      - opened
+      - synchronize
+      - ready_for_review
+
+# Cancel previous workflow run groups that have not completed.
+concurrency:
+  # Group workflow runs by workflow name, along with the head branch ref of the pull request
+  # or otherwise the branch or tag ref.
+  group: ${{ github.workflow }}-${{ github.event_name == 'pull_request' && github.head_ref || github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  phpunit:
+    runs-on: ubuntu-latest
+
+    continue-on-error: ${{ matrix.experimental }}
+    strategy:
+      fail-fast: false
+      matrix:
+        php: ['7.4', '8.0', '8.1']
+        wp: ['latest']
+        multisite: ['0', '1']
+        dependency-version: ['highest']
+        extensions: ['gd']
+        experimental: [false]
+        include:
+          # WP Trunk
+          - php: '7.4'
+            wp: 'trunk'
+            dependency-version: 'highest'
+            multisite: '0'
+            experimental: true
+          # PHP 8.0
+          - php: '8.0'
+            wp: 'latest'
+            dependency-version: 'prefer-stable'
+            multisite: '0'
+            experimental: false
+          # PHP 8.1
+          - php: '8.1'
+            wp: 'latest'
+            dependency-version: 'prefer-stable'
+            multisite: '0'
+            experimental: false
+          # PHP 8.1 / experimental
+          - php: '8.1'
+            wp: 'trunk'
+            dependency-version: 'highest'
+            multisite: '0'
+            experimental: true
+          # PHP 8.2 / experimental
+          - php: '8.2'
+            wp: 'trunk'
+            dependency-version: 'highest'
+            multisite: '0'
+            experimental: true
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Setup PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: ${{ matrix.php }}
+          tools: composer:v2
+          extensions: curl, date, dom, iconv, json, libxml, ${{ matrix.extensions }}
```

`extensions: ['gd']` is basically unused.
```suggestion
          extensions: curl, date, dom, iconv, json, libxml, gd
```